pkg/distributor/push_test.go

```diff
@@ -803,10 +802,10 @@ func TestHandler_DistributorPushErrorHTTPStatus(t *testing.T) {
 	userID := "user"
 	originalMsg := "this is an error"
 	originalErr := errors.New(originalMsg)
-	replicasNotMatchErr := distributorerror.NewReplicasDidNotMatch("a", "b")
-	tooManyClustersErr := distributorerror.NewTooManyClusters(10)
-	ingestionRateLimitedErr := distributorerror.NewIngestionRateLimited(10, 10)
-	requestRateLimitedErr := distributorerror.NewRequestRateLimited(10, 10)
+	replicasNotMatchErr := NewReplicasDidNotMatch("a", "b")
```

Comment:
We could actually unexport these `New` error calls. It would also make sense to add `Error` suffix to types and methods now.

Reply:
Thank you. Will fix it immediately